### ISSUE TITLE
Visited link colour contrast is too low as per WCAG Level AA

### ISSUE THREAD

**Issue by andy-polhill** [alphagov/govuk_frontend_toolkit#102]
I think the visited link colour may on occasion lack the necessary contrast to comply with WCAG Level AA guidelines. Anything 18px or above should be ok, but supporting paragraphs and smaller devices might be in violation. 

`$link-visited-colour: #2e8aca;`

>  The visual presentation of text and images of text has a contrast ratio of at least 4.5:1

Happy to create a pull request if you think appropriate, but I didn't want to mess with the branding.
